packages/server/lib/config.coffee

```diff
@@ -230,8 +230,10 @@ module.exports = {
     .value()
 
     if url = config.baseUrl
-      ## always strip trailing slashes
-      config.baseUrl = _.trimEnd(url, "/")
+      ## replace multiple slashes at the end of string to single slash
+      ## so http://localhost/// will be http://localhost/
+      ## https://regexr.com/48rvt
+      config.baseUrl = url.replace(/\/\/+$/, "/")
```

Comment:
regular expressions should be cached at the top of the file and dynamically generated... this also allows them to have a real name to provide additional context.

something like this... `multipleBackslashesRe`

also - whenever fixing bugs like this we need to add a comment at the top of the fix that points to the corresponding github issue to help developers understand **why** this change is necessary.

Reply:
@brian-mann will fix directly in dev. No action required here.